packages/widget-message-meet/src/containers/connected-activity-list/__snapshots__/index.test.js.snap

```diff
@@ -0,0 +1,4 @@
+exports[`ConnectedActivityList renders properly 1`] = `
+<div
+  className="activity-list" />
+`;
```

Comment:
Shouldn't this snapshot still have activities? Or does the mock store not have any?